### PR TITLE
Issue #SB-11433 feat: SSO Account Updater Job - Ignore roles in updates

### PR DESCRIPTION
**Pull request by JaikumarSRajan**

### CODE REVIEW

File: samza/sso-account-updater/src/main/java/org/sunbird/jobs/samza/service/SSOAccountUpdaterService.java [hunk 1]
```diff
@@ -160,42 +162,42 @@ private void validateName(Map<String, Object> eventMap, Map<String, Object> upda
...
-                    List<String> assignedRoles = (List<String>) organisation.get(SSOAccountUpdaterParams.roles.name());
-                    assignedRoles.remove(SSOAccountUpdaterParams.PUBLIC.name());
-                    if (CollectionUtils.isEqualCollection(passedRoles, assignedRoles)) {
                         isUserUpdateRequired = false;
-                        break;
-                    }
                 }
             }
 
+            if(existingRoles.isEmpty()) {
```
Comment (bvinayakumar) [inline samza/sso-account-updater/src/main/java/org/sunbird/jobs/samza/service/SSOAccountUpdaterService.java]: Format code.

File: samza/sso-account-updater/src/main/java/org/sunbird/jobs/samza/service/SSOAccountUpdaterService.java [hunk 2]
```diff
@@ -160,42 +162,42 @@ private void validateName(Map<String, Object> eventMap, Map<String, Object> upda
...
+            updateMap.put(SSOAccountUpdaterParams.organisations.name(), getOrganisationsListFromPayloadData(passedOrgId, userId, getPublicRole()));
         } else {
             boolean isUserUpdateRequired = true;
-
-            //ignore PUBLIC role for comparison
-            passedRoles.remove(SSOAccountUpdaterParams.PUBLIC.name());
+            Set<String> existingRoles = new HashSet<String>();
 
             for (Map<String, Object> organisation : organisations) {
+                List<String> assignedRoles = (List<String>) organisation.get(SSOAccountUpdaterParams.roles.name());
```
Comment (bvinayakumar) [inline samza/sso-account-updater/src/main/java/org/sunbird/jobs/samza/service/SSOAccountUpdaterService.java:176]: If the given roles list doesn't have PUBLIC, shouldn't we add it?

File: samza/sso-account-updater/src/main/java/org/sunbird/jobs/samza/service/SSOAccountUpdaterService.java (hunk 1 above)
Comment (JaikumarSRajan) [reply samza/sso-account-updater/src/main/java/org/sunbird/jobs/samza/service/SSOAccountUpdaterService.java]: Code formatting done

File: samza/sso-account-updater/src/main/java/org/sunbird/jobs/samza/service/SSOAccountUpdaterService.java (hunk 2 above)
Comment (JaikumarSRajan) [reply samza/sso-account-updater/src/main/java/org/sunbird/jobs/samza/service/SSOAccountUpdaterService.java:176]: The update/add member API takes care of adding PUBLIC

**Pull request merged**